dt-backend/src/model/user/user.controller.ts

```diff
@@ -0,0 +1,23 @@
+import {Body, Controller, Get, Post, Query} from "@nestjs/common";
+import { UserService } from "./user.service"
+import { CreateUserDto } from "./dto/create-user.dto"
+import {User as UserModel} from "../types/user.type";
+import {AuditLog} from "@prisma/client";
+import {Empire} from "../types/empire.type";
+
+
+@Controller("user")
+export class UserController {
+
+    constructor(private readonly userService: UserService) {}
+
+    @Post('userSignUp')
+    async signupUser(@Body() userData: { username: string; password: string; mail: string }): Promise<UserModel> {
+        return this.userService.createUser(userData);
+    }
+
+    @Get('user/:id')
```

Comment:
Remove `user/` from the route! The Routes stack onto each other!
Meaning the route is: `host:port/controller/route-path` in your current case that'd result into: `http://localhost:3000/user/user/someID`